mubench.reviewsite/tests/StoreMetadataTest.php

```diff
@@ -29,7 +29,7 @@ function test_store_metadata()
             "result" => "success",
             "runtime" => "42.1",
             "number_of_findings" => "23",
-            "detector" => $detector->id,
+            "-custom-stat-" => "-stat-val-",
```

Comment:
Why do you have to touch this test? Storing meta data has nothing to do with storing run info, has it?